GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Set and Get GVFS config settings.")]
+    public class ConfigVerb : GVFSVerb
+    {
+        private const string ConfigVerbName = "config";
+        private LocalGVFSConfig localConfig;
+
+        public override string EnlistmentRootPathParameter { get; set; }
+
+        protected override string VerbName
+        {
+            get { return ConfigVerbName; }
+        }
+
+        public override void Execute()
+        {
+            string[] args = Environment.GetCommandLineArgs();

```

Comment:
Please don't parse the command line args directly. Use our existing command line parsing patterns.